helpers/DynamoDB_insertTransformator.ipynb

```diff
@@ -0,0 +1,285 @@
+    "            'english': \"\"\"It seems that you have send {request_type} request. Right now we \\\n",
+    "don't support that type of functionality, but we plan to. ChatGPT will answer to your message. Please, notify us if \\\n",
+    "that was an error. That will improve the bot in your future requests.\"\"\"\n",
+    "        },\n",
+    "    'decline_reminders_ask':\n",
+    "        {\n",
+    "            'russian': 'Отменить уведомления и спросить ChatGPT напрямую',\n",
+    "            'english': 'Decline reminders and ask ChatGPT directly'},\n",
+    "    'ask_to_save':\n",
+    "        {\n",
+    "            'russian': 'Похоже, вы попросили сохранить что-то для вас, однако я пока не могу сохранять вещи. Это будет в будущих обновлениях. Пока я могу только устанавливать напоминания или отвечать на ващи вопросы.',\n",
+    "            'english': 'It sounds like you asked me to save something for you, however, I cant save things yet. That will be in future updates. For now I can only set reminders or answer your questions.'},\n",
+    "    'thank_you_for_error':\n",
+    "        {\n",
+    "            'russian': 'Спасибо что сообщили нам об ошибке! Мы постараемся учесть это в следующих обновлениях. Если это критично, вы можете связаться с создателем бота @plannerAI.',\n",
+    "            'english': 'Thanks for letting us know about the mistake! We will try to address this in our future updates. If this is critical, you can contact the creator of the bot @plannerAI.'},\n",
+    "    'it_was_mistake':\n",
+    "        {\n",
+    "            'russian': 'Сообщить об ошибке',\n",
+    "            'english': 'Report an error'},\n",
+    "}\n",
+    "\n",
+    "week_labels = {\n",
+    "    'Monday':\n",
+    "        {'russian': 'Понедельник',\n",
+    "         'english': 'Monday'},\n",
+    "    'Tuesday':\n",
+    "        {'russian': 'Вторник',\n",
+    "         'english': 'Tuesday'},\n",
+    "    'Wednesday':\n",
+    "        {'russian': 'Среда',\n",
+    "         'english': 'Wednesday'},\n",
+    "    'Thursday':\n",
+    "        {'russian': 'Четверг',\n",
+    "         'english': 'Thursday'},\n",
+    "    'Friday':\n",
+    "        {'russian': 'Пятница',\n",
+    "         'english': 'Friday'},\n",
+    "    'Saturday':\n",
+    "        {'russian': 'Суббота',\n",
+    "         'english': 'Saturday'},\n",
+    "    'Sunday':\n",
+    "        {'russian': 'Воскресенье',\n",
+    "         'english': 'Sunday'},\n",
+    "    'Today':\n",
+    "        {'russian': 'Сегодня',\n",
+    "         'english': 'Today'},\n",
+    "    'Tomorrow':\n",
+    "        {'russian': 'Завтра',\n",
+    "         'english': 'Tomorrow'},\n",
+    "}\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "90eca347",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def docts_converter(lable, days):\n",
```

Comment:
it is:

docs_converter or even label_converter, not "docts"
label, not lable

Reply:
Second: if you are working with multiple labels, why the first variable is in a single form?

Reply:
Third: it is always better to make a .copy() of original object. The reason is simple - if you pass object (in our case it is a json object) to the function and modify it, it can happen that original object will be modified too (that is called a memory-pointer variable, which points to the same spot in the memory in the original data and in the function to which it was passed)